main/src/test/scala/sbt/internal/TestBuild2.scala

```diff
@@ -0,0 +1,403 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt
+package internal
+
+import Def.{ ScopedKey, Setting }
+import sbt.internal.util.{ AttributeKey, AttributeMap, Relation, Settings }
+import sbt.internal.util.Types.{ const, some }
+import sbt.internal.util.complete.Parser
+import sbt.librarymanagement.Configuration
+
+import java.net.URI
+
+import hedgehog._
+import hedgehog.predef.sequence
+
+object TestBuild2 extends TestBuild2
```

Comment:
Sorry for the bad name. Is it worth renaming? That said, it looks like there aren't many places left that use `TestBuild` (`ParserSpec` and `Delegates`). Would you like me to convert them to hedgehog too if you're going to go ahead with this? I would be happy to do this whenever you like.

Reply:
I think if would be cool if we can migrate the other ones so we can remove the old TestBuild.